test/unit/visitor/defuse_analyze.cpp

```diff
@@ -445,6 +445,36 @@ SCENARIO("Perform DefUse analysis on NMODL constructs") {
         }
     }
 
+    GIVEN("Simple check of assigned variables") {
+        const std::string nmodl_text = R"(
+            NEURON {
+                GLOBAL x
+            }
+
+            ASSIGNED {
+                y
+            }
+
+            DERIVATIVE states {
+                x = 1
```

Comment:
Nitpick.. `x=1` could be removed from the `DERIVATIVE` block to make the MOD file valid for code generation?